examples/bri-3/src/bri/communication/agents/bpiMessagesStorage.agent.ts

```diff
@@ -51,20 +50,28 @@ export class BpiMessageStorageAgent extends PrismaService {
 
     return this.mapper.map(
       newBpiMessageModel,
-      getType<BpiMessage>(),
+      BpiMessage,
```

Comment:
again twice?

Reply:
this is because we are mapping from Prisma model to domain model